spec/Main.hs

```diff
@@ -130,9 +130,20 @@ larcenyFillTests = do
       let ctxt' = setRequest ctxt
                  $ (\(_,y) -> (requestWithUrl, y)) defaultFnRequest
       let s = view wpsubs ctxt'
-      let tpl = toTpl "<wp><wpPostByPermalink><wpDepartment><wpName /></wpDepartment></wp>"
+      let tpl = toTpl "<wp><wpPostByPermalink><wpDepartment><wpName /></wpDepartment></wpPostByPermalink></wp>"
       rendered <- evalStateT (runTemplate tpl [] s mempty) ctxt'
       rendered `shouldBe` "Sports"
+    it "should render stuff 2" $ do
```

Comment:
any chance we can improve naming here? :)

Reply:
lmao I forgot about that, the previous one should probably be changed from "should render stuff" as well